1528.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string} s
+ * @param {number[]} indices
+ * @return {string}
+ */
+var restoreString = function (s, indices) {
+  const sorted = indices.slice();
+  let result = [];
```

Comment:
```suggestion
  const result = [];
```
Use `const` unless you need to re-assign a new value to it